app/lib/shared/search_service.g.dart

```diff
@@ -33,6 +33,7 @@ PackageDocument _$PackageDocumentFromJson(Map<String, dynamic> json) =>
         dependencies: json['dependencies'] == null
             ? null
             : new Map<String, String>.from(json['dependencies'] as Map),
+        emails: (json['emails'] as List)?.map((e) => e as String)?.toList(),
```

Comment:
Oops! Looks like the generator didn't get run! @isoos

Reply:
https://github.com/dart-lang/pub-dartlang-dart/issues/812

Reply:
oops :(